frontend/src/pages/AdminPanel.js

```diff
@@ -0,0 +1,1253 @@
+                          className="h-12 w-12 object-cover rounded-full mx-auto"
+                        />
+                      ) : (
+                        <span className="text-gray-500 italic">Brak logo</span>
+                      )}
+                    </td>
+                    <td className="px-4 py-2">{kebab.name}</td>
+                    <td className="px-4 py-2">{kebab.address}</td>
+                    <td className="px-4 py-2 flex justify-center items-center space-x-4">
+                      <button
+                        className="text-blue-500 hover:text-blue-700"
+                        onClick={() => openKebabModal(kebab)}
+                      >
+                        <FontAwesomeIcon icon={faPen} />
+                      </button>
+                      <button
+                        className="text-red-500 hover:text-red-700"
+                        onClick={() => openKebabDeleteConfirm(kebab)}
+                      >
+                        <FontAwesomeIcon icon={faTrash} />
+                      </button>
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          </div>
+        )}
+        
+        {/* Tabela suggestions */}
+        {activeTab === 'suggestions' && (
+          <div className="w-full">
+            <h1 className="text-2xl font-bold text-darkGreen mb-6">Sugestie</h1>
+            <table className="w-full table-auto bg-white rounded-lg shadow-md">
+              <thead className="bg-darkGreen text-white">
+                <tr>
+                  <th className="px-4 py-2 text-center">Użytkownik</th>
+                  <th className="px-4 py-2 text-center">Tekst</th>
+                  <th className="px-4 py-2 text-center">Akcje</th>
+                </tr>
+              </thead>
+              <tbody>
+                {suggestions.map((suggestion) => (
+                  <tr key={suggestion.id} className="border-t text-center">
+                    <td className="px-4 py-2">{suggestion.user}</td>
+                    <td className="px-4 py-2">{suggestion.contents}</td>
+                    <td className="px-4 py-2 flex justify-center space-x-4">
+                      <button
+                        className="text-blue-500 hover:text-blue-700"
+                        onClick={() => openSuggestionModal(suggestion)}
+                      >
+                        <FontAwesomeIcon icon={faEye} />
+                      </button>
+                      <button
+                        className="text-red-500 hover:text-red-700"
+                        onClick={() => openSuggestionDeleteConfirm(suggestion)}
+                      >
+                        <FontAwesomeIcon icon={faTrash} />
+                      </button>
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          </div>
+        )}
+
+        {/* Panel edycji użytkownika */}
```

Comment:
I think all comments like that should be deleted, just for the code to be clearer

Reply:
nope, it all helps to navigate in code